src/endpoints/http/collections/delete_test.js

```diff
@@ -0,0 +1,82 @@
+import bodyParser from 'body-parser';
```

Comment:
if we separate out the business logic (delete collections) out of the controller, this test can be easily written without the need of supertest